scripts/run_lint.sh

```diff
@@ -18,6 +18,9 @@ function display_result {
   fi
 }
 
+./scripts/check_watch_statements.sh
+display_result $? 1 "Functional tests watch check"
```

Comment:
Not just watch now.

Reply:
Yeah, but I don't have a better name, and they all acheive a watch, so seems reasonable to me?